cookies.css

```diff
@@ -1,181 +1,146 @@
-    background-image: url('imgs/thumbs-up-solid-white.svg');
-  }
-
-  @media (min-width: 768px) {
-
-    div#cookies-modal.dark-theme {
-      background: rgba(0, 0, 0, 0.6);
-      border: 1px solid rgba(0, 0, 0, 0.8);
-    }
-
-  }
-
-  /*///////////// DARK MODE END /////////////*/
-
-  /*///////////// CUSTOMIZE TYPOGRAPHY START /////////////*/
-
-  div#cookies-modal-container, div#cookies-modal-container div, div#cookies-modal-container a, div#cookies-modal-container button {
-    font-family: 'Poppins', Helvetica, Arial, sans-serif;
-  }
-
-  /*///////////// CUSTOMIZE TYPOGRAPHY END /////////////*/
-
-  /*///////////// CUSTOMIZE COLORS START /////////////*/
-  
-  div#cookies-modal a, div#cookies-modal a:hover {
-    border-color: #bfa290;
-  }
-
-  div#cookies-modal button#accept-cookies-button {
-    background: #bfa290;
-  }
-
-  /*///////////// CUSTOMIZE COLORS END /////////////*/
-  
-  /*///////////// COOKIES MODAL END /////////////*/
\ No newline at end of file
+dialog#cookies-modal {
+	align-content: center;
+	align-items: center;
+	background: rgba(255, 255, 255, 0.8);
+	border-bottom: none;
+	border-left: none;
+	border-right: none;
+	border-top: 4px solid rgba(0, 0, 0, 0.18);
+	bottom: 0px;
+	flex-direction: row;
+	flex-wrap: wrap;
+	font-family: "Poppins", Helvetica, Arial, sans-serif;
+	justify-content: center;
+	margin: 0 auto;
+	max-width: 900px;
+	padding: 10px 20px;
+	top: auto;
+
+	&[open] {
+		display: flex;
+	}
+
+	&::backdrop {
+		backdrop-filter: blur(8px);
+	}
+
+	&.dark-theme {
+		background: rgba(0, 0, 0, 0.8);
+		border-top: 4px solid rgba(0, 0, 0, 1);
+
+		@media (min-width: 768px) {
+			background: rgba(0, 0, 0, 0.6);
+			border: 1px solid rgba(0, 0, 0, 0.8);
+		}
+
+		a,
+		a:hover,
+		header {
+			color: #fff;
+		}
+
+		button {
+			background: rgba(0, 0, 0, 0.4);
+			border-color: #00000047;
+			color: #fff;
+
+			&:hover {
+				background: rgba(0, 0, 0, 0.5);
+				color: #fff;
+			}
+
+			&#accept-cookies-button:before {
+				background-image: url("./imgs/thumbs-up-regular-white.svg");
+			}
+
+			&#accept-cookies-button:hover:before {
+				background-image: url("./imgs/thumbs-up-solid-white.svg");
+			}
+		}
+	}
+
+	@media (min-width: 768px) {
+		background: rgba(255, 255, 255, 0.65);
+		border-radius: 8px;
+		border: 1px solid rgba(0, 0, 0, 0.18);
+		bottom: 20px;
+		justify-content: space-between;
+		width: calc(100dvw - 80px);
+
+		footer {
+			margin: 0;
+		}
+	}
+
+	a {
+		border-bottom: 3px dotted #000;
+		color: #000;
+
+		&:hover {
+			border-bottom: 3px solid #000;
+			color: #000;
+		}
+
+		&,
+		&:hover {
+			border-color: #bfa290;
+		}
+	}
+
+	button {
+		background: rgba(255, 255, 255, 0.508);
+		border-radius: 4px;
+		border-width: 1px;
+		border: 2px solid #000;
+		color: #000;
+		float: right;
+		font-size: 0.9rem;
+		height: 36px;
+		letter-spacing: 0;
+		opacity: 0.8;
+		padding: 5px 25px 4px;
+		text-transform: uppercase;
+
+		&:first-child {
+			margin-left: 6px;
```

Comment:
I didn't changed the styles much other than modernizing them, but I would recommend using `gap` instead of setting a hardcoded margin.